dnf-behave-tests/features/vars-releasever.feature

```diff
@@ -86,3 +86,63 @@ Scenario: Releasever is substituted in baseurl via a value detected from 'system
     And Transaction is following
         | Action        | Package                       |
         | install       | setup-0:2.12.1-1.fc29.noarch  |
+
+@destructive
+Scenario: Releasever is substituted in baseurl using vars loaded from location (host or installroot) like used repos
```

Comment:
The scenario name doesn't sound right to me. What about
Releasever is substituted in baseurl using vars loaded from the same location (host or installroot) as repos